dist/tools/pyterm/pyterm

```diff
@@ -729,6 +738,12 @@ if __name__ == "__main__":
                         action="store_true",
                         help="toggles the DTR and RTS pin of the serial line "
                         "when connecting, default is not toggling the pins")
+    parser.add_argument("-ir", "--invert_RTS",
+                        action="store_true",
+                        help="inverts the normal logic state of RTS")
```

Comment:
maybe change the message to "inverts RTS logic state"? Same for DTR

Reply:
Done!

Reply:
The options have been changed now. Please check that.